circle.yml

```diff
@@ -1,11 +1,18 @@
 machine:
   xcode:
-    version: 8.0
+    version: 7.3
   environment:
     NSUnbufferedIO: YES
 
 dependencies:
   override:
+    - gem install cocoapods
+    - pod install
```

Comment:
Might be nice to have these in a `bin/setup` so contributors can run the same setup locally